velox/functions/prestosql/DecimalArithmetic.cpp

```diff
@@ -162,6 +162,50 @@ class Subtraction {
   }
 };
 
+class Multiply {
+ public:
+  template <typename R, typename A, typename B>
+  inline static void
+  apply(R& r, const A& a, const B& b, uint8_t aRescale, uint8_t bRescale) {
+    try {
+      r.setUnscaledValue(checkedMultiply<int128_t>(
```

Comment:
try-catch introduced in-efficiency. Can we directly use `bool overflow = __builtin_mul_overflow(a, b, &result);` ?

Reply:
Just leave the exception uncatched should be ok, that's what other types do.